src/Menu/Menu.php

```diff
@@ -50,7 +50,10 @@ public function getItems(): array
 
 	private function registerGroup(?string $group): string
 	{
-		$group = $group ?: 'default';
+		$group = is_string($group) === false
+			? 'default'
+			: $group;
```

Comment:
Toto není ekvivalentní náhrada. Elvis operátor zde sloužil k ověření, že nejde o `null` a prázdný string.

Více info: https://php.baraja.cz/ternarni-operator#overeni-prazdnosti-hodnoty

Možné řešení:

```suggestion
		$group = $group !== null && $group !== '' ? $group : 'default';
```